nl/guides/course-management/README.md

```diff
@@ -0,0 +1,417 @@
+`Registratieverzoeken`|   ![image](../../../images/staff_registration_icons/approve.png)|     goedkeuren van het [registratieverzoek](#registratieverzoek) dat door de gebruiker werd [ingediend](/nl/for-students#registratieverzoek-indienen), waardoor de gebruiker [geregistreerd](/nl/for-students#cursus-registreren) wordt voor de cursus|
+`Registratieverzoeken`|   ![image](../../../images/staff_registration_icons/decline.png)|      afkeuren van het [registratieverzoek](#registratieverzoek) dat door de gebruiker werd [ingediend](/nl/for-students#registratieverzoek-indienen), waardoor de gebruiker [uitgeschreven](/nl/for-students#cursus-uitschrijven) wordt uit de cursus|
+
+### Cursusbeheerders aanduiden
+
+<h1 id="cursusbeheerder"></h1>
+
+Een cursusbeheerder is een
+[geregistreerde](/nl/for-students#cursus-registreren)
+[cursusgebruiker](#cursusgebruiker) met
+beheersrechten voor de [cursus](#cursus).
+Een [lesgever](/nl/user-management#lesgever) die een nieuwe
+cursus aanmaakt wordt er automatisch voor geregistreerd en wordt er
+meteen ook de eerste cursusbeheerder van. Lesgever hebben voor het
+beheren van cursussen echter geen bijkomende rechten ten opzichte van
+andere cursusbeheerders en kunnen ook
+geen cursussen beheren waarvoor ze geen cursusbeheerder zijn.
+
+In de tab `Geregistreerd` van het [overzicht](#cursusgebruikersoverzicht)
+met alle [cursusgebruikers](#cursusgebruiker) kan je de cursusbeheerders herkennen aan het icoontje in de
+linkermarge.
+
+![image](./staff.course_users_admin.png)
+
+In de tab kan je de volgende actieknoppen gebruiken om cursusbeheerders
+aan te duiden:
+
+|knop                                                              |actie                                                                                |
+|------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+![image](../../../images/staff_registration_icons/make_course_admin.png)    |cursusgebruiker zonder beheersrechten voor de cursus promoveren tot cursusbeheerder  |
+![image](../../../images/staff_registration_icons/make_student.png)         |cursusbeheerder degraderen tot cursusgebruiker zonder beheersrechten voor de cursus  |
+
+::: tip Belangrijk
+
+Een [cursusbeheerder](#cursusbeheerder)
+die zich [uitschrijft](/nl/for-students#cursus-uitschrijven) uit een [cursus](#cursus)
+verliest zijn status van cursusbeheerder.
+
+Om ervoor te zorgen dat een cursus altijd minstens één cursusbeheerder
+heeft, kan de laatste cursusbeheerder zich niet uitschrijven en kan hij
+zichzelf ook niet degraderen tot cursusgebruiker zonder beheersrechten
+voor de cursus.
+
+Na het aanmaken van een cursus kan een lesgever zich uitschrijven als
+hij andere cursusbeheerders aangeduid heeft. Hij kan door andere
+cursusbeheerders ook gedegradeerd worden tot cursusgebruiker zonder
+beheersrechten voor de cursus.
+:::
+
+## Navigeren naar oplossingen
```

Comment:
This section should contain 3 clear parts:
- Viewing all exercises of a course
- Viewing all exercises of a certain user
- Viewing all exercises of a certain exercise (from course page and from exercise page)